calibr8/tests.py

```diff
@@ -2,6 +2,7 @@
 import datetime
 import logging
 import pathlib
+from msilib.schema import Class
```

Comment:
```suggestion
```

Probably got inserted by some autocompletion.
I recommend staging lines in VS Code manually, because then you notice such things before the commit.